Added support for Address Fields in Backend & Database

alkatra opened the pull request:
Completed backend support for address fields in database, as well as removed "required" tab from address line 2 field and added backend validation for postcode in range [1000:9999]

alkatra committed:
Added support for Address Fields in Backend & Database

Completed backend support for address fields in database, as well as removed "required" tab from address line 2 field and added backend validation for postcode in range [1000:9999]

SDzartov commented:
Address data needs to be in it's own table under the user.db and linked using the same primary key as the user table.

alkatra committed:
Moved Address fields to Separate Table

Separate Address table created with Foreign Key being username from User table.

alkatra commented:
Done @SDzartov

SDzartov commented:
you need to rebase from develop

SDzartov committed:
Merge branch 'develop' into feature_FLASK_address_fields

Signed-off-by: MoreDread <43129046+SDzartov@users.noreply.github.com>
